rsp_game/lib/result_printer.rb

```diff
@@ -1,34 +1,26 @@
-class ResultPrinter
-  WINNING_COMBINATIONS_FOR_USER = [ ['r', 's'], ['s', 'p'], ['p', 'r'] ].freeze
+module Game::ResultPrinter
+  WINNING_COMBINATIONS_FOR_USER = [%w[r s], %w[s p], %w[p r]].freeze
 
   attr_accessor :result
 
-  def initialize(game)
-    @game = game
+  def check_result(game)
+    self.result = if game.user_value == game.computer_value
```

Comment:
Please refactor this.
Maybe you can add method where you can check statements

Reply:
I refactored it, please check